### ISSUE TITLE
Lag of control samsung tizen tv on Home Assistant

### ISSUE THREAD

**Issue by salmahassan7979** [ollo69/ha-samsungtv-smart#80]
ps anyone else find it’s probably around 10 seconds from changing source before it shows the new source in HA etc? Anyway to speed this up?

_Originally posted by @mbarnes106 in https://github.com/ollo69/ha-samsungtv-smart/issues/35#issuecomment-756789768_

**Comment by ollo69** [comment]
This is not an issue. This is a component based on device and cloud poll, polling have interval. We cannot poll the TV and SmartThings every or your network and HA will stop working.

**Comment by salmahassan7979** [comment]
Thank you for answering me but I can't understand it, if i use Smartthings app it doesn't show any delays but if i use HA i wait about 10secs for the action to happen on TV

**Comment by ollo69** [comment]
We are not Samsung and don't know what Samsung do inside his app. Anyway i you any have valid solution that will not break HA state machine feel free to suggest it here or open a PR with changes, otherwise please close this issue.

**Comment by WibiNorg** [comment]
I'm having the same problem. Coming from Homebridge, the app-state changes instantly, but with this HASS-intergration it takes almost 10 seconds to update the app status and therefore making proper automations based on which app is loaded useless.

For instance: 
I had my TV intergrated in Homekit through HomeBridge via a Raspberry Pi 4.
I've added input-booleans linked to a couple apps, so a input-boolean would appear in my Homekit, example: "TV on Netflix". 
Then I set up an automation where whenever that input-boolean was turned ON, the lights would go to a certain scene. This worked absolutely perfect. 

Is there any way to get this kind of pinging-state intergrated in this? It would be amazing if I could intergrate all my smarthome stuff in 1 platform, since I now still have to use the HomeBridge SamsungTizen app just for this neat automation.
